ImageSegmentation/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
This file is all commented out, maybe remove this file and remove it from build.gradle?